components/clm/src/main/ncdio_pio.F90.in

```diff
@@ -501,11 +501,11 @@ contains
        if (ier == PIO_NOERR) nj = 1
     end if
     
-    ier = pio_inq_dimid (ncid, 'topounit', dimid)
-    if (ier == PIO_NOERR) then
-       ier = pio_inq_dimlen(ncid, dimid, ni)
-       if (ier == PIO_NOERR) nj = 1
-    end if       
+    !ier = pio_inq_dimid (ncid, 'topounit', dimid)
```

Comment:
Remove the commented out code

Reply:
Thanks! Commented code has been removed.